### PR TITLE
Services g

### PR DESCRIPTION
**Pull request by giedrunas**
6 x service items (js) (icon, h3, p)
add style (+ background color)

### CODE REVIEW

File: js/data/services.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        paragraph:'The standard chunk of Lorem Ipsum used since the is reproduced below for those interested.',
+
+    },
+    {
+        icon: "mbri-edit",
+        header: 'Easy to customize',
+        paragraph:'The standard chunk of Lorem Ipsum used since the is reproduced below for those interested.',
+
+    }
+]
```
Comment (UrteZ) [inline js/data/services.js:38]: reikia prideti ;

File: css/layout.css [hunk 2]
```diff
@@ -69,7 +69,41 @@ body {
             }
 
         /* services */
-
+    /* reikia sutvarkyti responsive */
```
Comment (Karina-N) [inline css/layout.css:72]: Kai susiaureja ekranas ir viskas sukrenta i viena stulpeli, dar reiktu ir sucentruoti

**Review by Karina-N — APPROVED** [review]
Responsiveness looks way better after fixes, well done

**Review by UrteZ — APPROVED** [review]

**Review by mildasuziedelyte — APPROVED** [review]
well done!

**Review by ugniusnor — APPROVED** [review]

**Pull request merged**